lineman/app/services/form_service.coffee

```diff
@@ -41,6 +41,7 @@ angular.module('loomioApp').factory 'FormService', ($rootScope, FlashService, $f
         scope.isDisabled = false
 
     submit: (scope, model, options = {}) ->
+      DraftService.applyDrafting(scope, model) if options.allowDrafts
```

Comment:
This makes the interface for using drafts dead easy. If we're using the FormService, all we need to do is pass the `allowDrafts: true` option, and the DraftService will take care of the rest.